composer.json

```diff
@@ -24,7 +24,7 @@
         "imagine/imagine": "~0.6.1",
         "liip/theme-bundle": "1.3.*",
         "massive/search-bundle": "dev-develop",
-        "sulu/document-manager": "dev-develop",
+        "sulu/document-manager": "dev-query_builder_final",
```

Comment:
This is probably wrong now, you've renamed the branch to something using 2016, didn't you?